app/grandchallenge/evaluation/models.py

```diff
@@ -1207,6 +1230,33 @@ class PhaseGroupObjectPermission(GroupObjectPermissionBase):
     content_object = models.ForeignKey(Phase, on_delete=models.CASCADE)
 
 
+class PhaseAlgorithmInterface(models.Model):
+    phase = models.ForeignKey(Phase, on_delete=models.CASCADE)
+    interface = models.ForeignKey(AlgorithmInterface, on_delete=models.CASCADE)
+    is_default = models.BooleanField(default=False)
```

Comment:
Why do we need a default algorithm interface for Phases? I thought this was only used to select a default for the "Try out algorithm" page. Do we even use it there?

Reply:
Removed